src/pages/Main/Main.js

```diff
@@ -1,8 +1,70 @@
 import React, { Component } from 'react';
+
+import { CardList } from './Card-list';
+import Clock from './Clock';
+import Nav from '../../components/Nav/Nav';
+import Menu from './Menu';
+import Banner from './Banner';
 import './Main.scss';
 
 export default class Main extends Component {
+  constructor() {
+    super();
+    this.state = {
+      monsters: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(users => this.setState({ monsters: users }));
+  }
+
   render() {
-    return <div>Hello, Main!</div>;
+    return (
+      <>
+        <Nav />
+
+        <div className="search-bar">
+          <input
+            className="search"
+            type="search"
+            placeholder="검색어를 입력해주세요."
+          />
+        </div>
+
+        <Banner />
+
+        <Menu />
+
+        <CardList monsters={this.state.monsters} />
+
+        <div className="time-section">
```

Comment:
```suggestion
        </div>
        <Banner />
        <Menu />
        <CardList monsters={this.state.monsters} />
        <div className="time-section">
```
불필요하게 한줄씩 띄울 필요 없습니다!

Reply:
> 불필요하게 한줄씩 띄울 필요 없습니다!

넵 확인했습니다 반영하겠습니다